drift_dev/lib/src/writer/manager/manager_templates.dart

```diff
@@ -0,0 +1,387 @@
+  String rootTableManagerGetter(
+      DriftTable table, String rootTableManagerClass) {
+    return '$rootTableManagerClass get ${table.dbGetterName} => $rootTableManagerClass(_db, _db.${table.dbGetterName});';
+  }
+
+  /// Returns the name of the root manager class for a table
+  ///
+  /// One of these classes is generated for each table in the database
+  ///
+  /// E.g. `\$UserTableManager`
+  String rootTableManagerName(DriftTable table) {
+    return '\$${table.entityInfoName}TableManager';
+  }
+
+  /// Returns the name of the manager class for a table
+  ///
+  /// When using modular generation the manager class will contain the correct prefix
+  /// to access the table manager
+  ///
+  /// E.g. `i0.UserTableTableManager` or `\$UserTableTableManager`
+  String rootTableManagerWithPrefix(DriftTable table, TextEmitter leaf) {
+    return leaf
+        .dartCode(leaf.generatedElement(table, rootTableManagerName(table)));
+  }
+
+  /// Returns the name of the processed table manager class for a table
+  ///
+  /// This does not contain any prefixes, as this will always be generated in the same file
+  /// as the table manager and is not used outside of the file
+  ///
+  /// E.g. `$UserTableProcessedTableManager`
+  String processedTableManagerName(DriftTable table) {
+    return '\$${table.entityInfoName}ProcessedTableManager';
+  }
+
+  /// Class which represents a table in the database
+  /// Contains the prefix if the generation is modular
+  /// E.g. `i0.UserTable`
+  String tableClassWithPrefix(DriftTable table, TextEmitter leaf) =>
+      leaf.dartCode(leaf.entityInfoType(table));
+
+  /// Class which represents a row in the table
+  /// Contains the prefix if the generation is modular
+  /// E.g. `i0.User`
+  String rowClassWithPrefix(DriftTable table, TextEmitter leaf) =>
+      leaf.dartCode(leaf.writer.rowType(table));
+
+  /// Name of this tables filter composer class
+  String filterComposerNameWithPrefix(DriftTable table, TextEmitter leaf) {
+    return leaf
+        .dartCode(leaf.generatedElement(table, filterComposerName(table)));
+  }
+
+  /// Name of this tables filter composer class
+  String filterComposerName(
+    DriftTable table,
+  ) {
+    return '\$${table.entityInfoName}FilterComposer';
+  }
+
+  /// Name of this tables ordering composer class
+  String orderingComposerNameWithPrefix(DriftTable table, TextEmitter leaf) {
+    return leaf
+        .dartCode(leaf.generatedElement(table, orderingComposerName(table)));
+  }
+
+  /// Name of this tables ordering composer class
+  String orderingComposerName(DriftTable table) {
+    return '\$${table.entityInfoName}OrderingComposer';
+  }
+
+  /// Name of the typedef for the insert companion builder for a table
+  ///
+  /// This is the name of the typedef of a function that creates new rows in the table
+  String insertCompanionBuilderTypeDef(DriftTable table) {
+    return '\$${table.entityInfoName}InsertCompanionBuilder';
+  }
```

Comment:
Are these typedefs possibly referenced for other tables? In that case we might have to add a `.generatedElement` variant as well

Reply:
These are only used by their own table managers, no prefix is required